recipe/meta.yaml

```diff
@@ -15,7 +15,11 @@ build:
   number: 0
   script: {{ PYTHON }} -m pip install . -vv
```

Comment:
You don't need anything below this line

Reply:
Any script lines that is

Reply:
I gave it a go for giggles, but we do a lot of cross-compiling Rust targets and other things during the build process to generate these wheels on PyPi; that is why I'm trying to install them directly from there, rather than maintaining two build systems.

Reply:
and again, the previous setup works for all platforms aside from ppc64le, which works fine in the GitHub actions CI.